client/app/components/viz/VariantInspectCard.vue

```diff
@@ -1248,6 +1352,16 @@ export default {
           }
         },2000)
       }
+      
+      if (self.$refs.popupPedigreeGenotypeViz) {
+        self.$refs.popupPedigreeGenotypeViz.update();
```

Comment:
I am wondering why we have to call this update() method here. Is there a way to structure the code so any reactivity is just built in automatically?

Reply:
Checked for both of the pedigree charts (popup and regular) and it seems to render just fine without this code. I had to change my popup logic just a little to get it to render without this call but it is working. 

The only thing to consider would be if you expect the updates in the pedigreeGenotypeData to be nested down in the object... the vue documentation says that this can be tricky and you might need a "deep" watcher for that. But if you are okay taking out this code we can it is commented out for now let me know.